applications/GeoMechanicsApplication/custom_utilities/input_utility.h

```diff
@@ -0,0 +1,30 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#pragma once
+
+#include <string>
```

Comment:
Shouldn't we now include `<filesystem>` rather than `<string>`?
```suggestion
#include <filesystem>
```

Reply:
Removed